src/db.ts

```diff
@@ -0,0 +1,70 @@
+import util from 'util';
+import AWS from 'aws-sdk';
+
+export class DynamoDB {
+  client: AWS.DynamoDB.DocumentClient;
+  queryDB: any;
+  putDB: any;
+  scanDB: any;
+  getDB: any;
```

Comment:
I think we can at least go for `(any) => any` for the function type vs just `any`